src/community/resilience.jl

```diff
@@ -0,0 +1,111 @@
+#=
+Some functions from *Universal resilience patterns in complex networks*
+by Gao et al. (2016)
+=#
+
+using StatsBase: mean, std
```

Comment:
this should be coming from `Statistics`, not `StatsBase`

Reply:
Done. But is StatsBase not the default one for such functions?